common/util.go

```diff
@@ -48,3 +52,10 @@ func GetPostgresDataDir(dataDir string) string {
 func GetPostgresSocketDir() string {
 	return "/tmp"
 }
+
+// DefaultMigrationVersion returns the default migration version string.
+// Use the concatenation of current time and uuid to guarantee uniqueness in a monotonic increasing way.
+// We cannot add task ID because tenant mode databases should use the same migration version string when applying a schema update.
+func DefaultMigrationVersion() string {
+	return fmt.Sprintf("%s.%s", time.Now().Format("20060102150405010203"), uuid.New())
```

Comment:
I am wondering why we prefer microsecond + uuid, instead of

1. second + uuid
2. nanosecond

Reply:
Switched to second + uuid.

I noticed still receiving conflict errors upon using nanoseconds and decided to append an uuid suffix.

Reply:
Are we able to specify the version string from the test?

Reply:
Nah, since we're simulating from end-user perspective.

Reply:
My understanding is this is likely the problem only happens in the test because it's run much more frequently?

But here we are adding an extra suffix, so we are essentially changing a global behavior to address a test issue?

Reply:
After a second thought, I think you might be right. Changed tests to sleep 2 seconds between issues.